tests/processes/ProcessingTest.py

```diff
@@ -347,39 +347,6 @@ def test_create_process_group(self):
             # python modules subprocess and os
             self.assertEqual(p.pid, pgid)
 
-    def test_filter_raising_callables(self):
```

Comment:
Why is this removed?

Reply:
It would have gotten too complicated keeping this as a dedicated function. It aimed to be simple, expecting a fixed set of arguments for functions/classes being invoked. But with the new debugger changes, we instantiate bears differently, depending on whether we have switched on debugging for them or not. I recommended @RaiVaibhav to inline it, so future modifications are easier + better to understand.

Reply:
help me in getting `inline` I moved it into separate commit :sweat_smile:  (english problem sorry)